infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -731,7 +731,7 @@ private void onStandbyClusterAddRemove(TopologyDescriptor discoveredTopology) {
     private boolean processDiscoveredTopology(TopologyDescriptor topology, boolean update) {
         // Health check - confirm this node belongs to a cluster in the topology
         if (topology != null && clusterPresentInTopology(topology, update)) {
-            log.info("Node[{}/{}] belongs to cluster, descriptor={}, topology={}", localEndpoint, localNodeId, localClusterDescriptor, topology);
+            log.info("Node[{}/{}] belongs to cluster", localEndpoint, localNodeId);
```

Comment:
they are from different branches, one is belongs to and another one is does not belong to

Reply:
I don't think you should remove the descriptor and local descriptor info. If you see `processDiscoveredTopology` is called from 2 different paths, and on one of them you do not have info of the topology descriptor. This is critical to understand which role we took. In any case you could remove the topology info being printed from processTopologyChangeNotification as it will be printed in here. Nevertheless, this event happens rarely so its not really noisy.